app/models/host/managed.rb

```diff
@@ -2,7 +2,12 @@ class Host::Managed < Host::Base
   include Hostext::Search
   include SelectiveClone
   include HostParams
-  PROVISION_METHODS = %w[build image]
+
+  class_attribute :provision_methods
+  self.provision_methods = {
+    'build' => 'Network Based',
```

Comment:
Use `N_('Network Based')` to extract the strings without translating them (you're translating them correctly later with `_()` so that's good).